### ISSUE TITLE
Submit IAAF short description - Due Sept 15. 

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2825]
We are excited for your upcoming IAAF presentation in November! Details for your presentation and next steps are below. If this block of time does not work for you, please let us know by September 8th.

We will share your room location, presenter guidance, and a final presentation date and time in the coming weeks. We appreciate your patience as we finalize the schedule. Each presenter will also be asked to attend a practice session in the weeks leading up to the event - information about those dates will be sent out as soon as possible. Please reach out to [IAAF@gsa.gov](mailto:IAAF@gsa.gov) with any questions in the meantime.

Presentation date and time: Tentatively scheduled for Thursday, November 9th between 12:00 p.m. and 3:00 p.m.

Session type: Presentation

Duration: 45 minutes

Location: In-person at FDIC Virginia Square (3501 Fairfax Drive, Arlington VA)

Suggested Modifications/Notes:

1.Consider providing a specific example where accessibility and cultural sensitivity has made a major difference for ONRR employees with disabilities and/or members of the public whom you serve (possible example - language and formatting that improved communication with a tribal nation).

Next Steps:

Please email a 2-3 sentence session description for the program agenda to [IAAF@gsa.gov](mailto:IAAF@gsa.gov) by Friday, September 15th. Examples of previous agenda descriptions are available in the [IAAF Archives](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.section508.gov%2Fiaaf%2Farchives%2Fagenda-2022%2F&data=05%7C01%7CLindsay.Goldstein%40onrr.gov%7C277e30bd9fdd4f743d9a08dbaef6ea7f%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C638296147751621940%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BAWUqkil3wjIV%2BIaxvuaBp4ZM3bkvdi3uKmr0PfXrCY%3D&reserved=0).

Please submit the below items by Friday, September 15th at 4:00 p.m. EST. via the [Google form](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.google.com%2Fforms%2Fd%2Fe%2F1FAIpQLSfSRCUNH3zB1YONFynQ0P0Dfuh5BZnkB4HxiiTWMZ5quw0RuA%2Fviewform&data=05%7C01%7CLindsay.Goldstein%40onrr.gov%7C277e30bd9fdd4f743d9a08dbaef6ea7f%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C638296147751621940%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=1AEMN%2FtlfgAjzhpQ3hNGRY05Su%2FW1%2FS331MW6zw7wo8%3D&reserved=0) (or email the items to [IAAF@gsa.gov](mailto:IAAF@gsa.gov) if needed). If you have co-presenters or are on a panel, each presenter should submit their own form with:

A professional headshot

A brief professional biography

Signed photo release/gratuitous services form

Please submit your presentation slides using the attached template by emailing it to [IAAF@gsa.gov](mailto:IAAF@gsa.gov) by Friday, September 29th, 2023. If your slides will be done by September 15th, feel free to submit them via the Google Form with the bio, headshot, and presenter form. Please follow the included instructions to ensure the presentation is accessible.

Please note: You are responsible for getting approval from your agency to present and to clear the content of your presentation.

**Comment by Maroyafaied** [comment]
@lpgoldstein what do you think of using this for the session description? It's from the proposal we submitted and made minor changes. 
"This session will shed light on how a small agency has embedded accessibility into its core values and practices. By prioritizing accessibility and cultivating a culture where creating accessible content is the responsibility of every individual, we have made significant strides towards inclusivity. We hope to inspire and provide insights that can be applied in your own organizations, fostering a culture of accessibility and inclusivity."

**Comment by lpgoldstein** [comment]
@Maroyafaied Looks great, thank you. I just edited the last sentence so it all now reads "This session will shed light on how a small agency has embedded accessibility into its core values and practices. By prioritizing accessibility and cultivating a culture where creating accessible content is the responsibility of every individual, we have made significant strides towards inclusivity. We hope to inspire and provide insights that can be applied in your own organizations to foster a culture of accessibility and inclusivity."
